### DISCUSSION TITLE
How do I prevent LSP signatures from popping up?

### DISCUSSION THREAD

**Discussion by givensuman** [LazyVim/LazyVim#3993] [Q&A] [answered]
At present when editing an object in my web project I get a big popup describing some LSP info.

![image](https://github.com/LazyVim/LazyVim/assets/16063606/7c381808-9513-429a-8615-4b6dda89ad49)

It's covering up some autocomplete information which is irritating. Ideally it would only show up if I prompt it to with `<leader>K`

Any ideas? I've disabled my plugins so I'm pretty sure this is built-in LazyVim settings, but I'm not sure which plugin to tweak

**Comment by dpetka2001** [comment, marked as answer]
It's `noice.nvim`. Check its docs for `lsp.signature` to disable it.

**Reply by givensuman** [reply]
Ahh thanks.

```lua
return {
  "folke/noice.nvim",
  opts = {
    lsp = {
      signature = {
        enabled = false,
      },
    },
  },
}

``` 
fixes it
